src/doc/src/reference/registry-web-api.md

```diff
@@ -307,6 +307,7 @@ A successful response includes the JSON object:
 ```javascript
 {
     // Indicates the remove succeeded, always true.
+    "message": "Needed but not used yet!",
```

Comment:
A few points here:

* This moves the comment to the wrong line.
* The field key is `"msg"`.
* Each key should have a comment that explains what it is. The explanation shouldn't go in the string.  I would just explain it as something like "This field is currently ignored by cargo." or something like that.

FWIW, it also seems like it would be trivial to change cargo to not ignore it.

Reply:
Thanks for the suggestions, some revisions to it have been made.
Do let me know what you think about it!